homeassistant/components/abode/camera.py

```diff
@@ -30,7 +30,7 @@ async def async_setup_entry(
     data: AbodeSystem = hass.data[DOMAIN]
 
     async_add_entities(
-        AbodeCamera(data, device, TIMELINE.CAPTURE_IMAGE)
+        AbodeCamera(data, device, TIMELINE.CAPTURE_IMAGE)  # pylint: disable=no-member
```

Comment:
There indeed doesn't seem to be a `CAPTURE_IMAGE` here: <https://github.com/jaraco/jaraco.abode/blob/main/jaraco/abode/helpers/timeline.py>
Is the library dynamically adding it or something?

Reply:
Correct, the symbols are being dynamically exported. See https://github.com/jaraco/jaraco.abode/blob/4bd291291222b9689de7c815997f8e643ca1ec23/jaraco/abode/helpers/timeline.py#L81-L87